package.json

```diff
@@ -4,7 +4,11 @@
   "description": "Probation task for Andersen, written on native JavaScript",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "start": "webpack-dev-server --open",
+    "lint": "./node_modules/.bin/eslint src/**/*.js",
```

Comment:
`eslint src/**/*.js`, there is no need to to write full path to node_module bin file